lib/systemd/system-sleep/pop-default-settings_bluetooth-suspend

```diff
@@ -2,15 +2,19 @@
 
 set -e
 
-#TODO: save rfkill state?
 case "$2" in
     suspend | hybrid-sleep)
         case "$1" in
             pre)
-				rfkill block bluetooth
+                if [ $(cat /var/lib/systemd/rfkill/*bluetooth) -eq 1 ]; then
+                    > /home/$USER/blocked.bt
```

Comment:
The location for this file... seems odd.  First it is not clear that `$USER` is defined within the context of systemd.  Moreover, it seems kinda hacky to place it a users home directory.   Since this is a more system-level state variable, it might be better to put it in `/run`

Reply:
I agree this is a better location.